packages/core/admin/admin/src/content-manager/utils/validation.ts

```diff
@@ -1,458 +1,314 @@
+           */
+          if (!value || (typeof value === 'string' && value.length === 0)) {
+            return true;
           }
 
-          break;
-        }
+          try {
+            JSON.parse(value);
 
-        case 'max': {
-          if (type === 'biginteger') {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.isInferior(errorsTrads.max, validationValue);
-          } else {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.max(validationValue, errorsTrads.max);
+            return true;
+          } catch (err) {
+            return false;
           }
-          break;
         }
-        case 'maxLength':
-          // @ts-expect-error – see comment at top of file
-          schema = schema.max(validationValue, errorsTrads.maxLength);
-          break;
-        case 'min': {
-          if (type === 'biginteger') {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.isSuperior(errorsTrads.min, validationValue);
-          } else {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.min(validationValue, errorsTrads.min);
-          }
-          break;
-        }
-        case 'minLength': {
-          if (!options.isDraft) {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.min(validationValue, errorsTrads.minLength);
-          }
-          break;
-        }
-        case 'regex':
-          // @ts-expect-error – see comment at top of file
-          schema = schema.matches(new RegExp(validationValue), {
-            message: errorsTrads.regex,
-            excludeEmptyString: !validations.required,
-          });
-          break;
-        case 'lowercase':
-          if (['text', 'textarea', 'email', 'string'].includes(type)) {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.strict().lowercase();
-          }
-          break;
-        case 'uppercase':
-          if (['text', 'textarea', 'email', 'string'].includes(type)) {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.strict().uppercase();
-          }
-          break;
-        case 'positive':
-          if (isFieldTypeNumber(type)) {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.positive();
-          }
-          break;
-        case 'negative':
-          if (isFieldTypeNumber(type)) {
-            // @ts-expect-error – see comment at top of file
-            schema = schema.negative();
-          }
-          break;
-        default:
-          schema = schema.nullable();
-      }
-    }
-  });
-
-  return schema;
+      );
+    case 'password':
+    case 'richtext':
+    case 'string':
+    case 'text':
+      return yup.string();
+    case 'uid':
+      return yup.string().matches(/^[A-Za-z0-9-_.~]*$/);
+    default:
```

Comment:
Could we then remove the case for 

```
    case 'date':
    case 'datetime':
    case 'time':
    case 'timestamp':

```

As all it does is apply yup.mixed() ?

Reply:
yes! good idea